apps/cookbook/src/app/showcase/default-list-showcase/default-list-showcase.html

```diff
@@ -0,0 +1,102 @@
+  <code>&lt;ul&gt;</code> or <code>&lt;ol&gt;</code>, and a padding of 12px to the right of every
+  individual bullet or number in the <code>&lt;li&gt;</code> element(s). Lastly it adds a margin of
+  8px to the top of every <code>&lt;li&gt;</code>, except for the first
+  <code>&lt;li&gt;</code> element within the <code>&lt;ul&gt;</code> or <code>&lt;ol&gt;</code>.
+</p>
+<h2>Unordered List Example</h2>
+<ul>
+  <li>Aenean id odio turpis</li>
+  <li>Lorem ipsum dolor sit amet</li>
+  <li>Donec ullamcorper, risus id venenatis</li>
+</ul>
+<h2>Unordered List /w Nested Lists Example</h2>
+<ul>
+  <li>Maecenas purus sapien</li>
+  <li>Pellentesque ultricies mi</li>
+  <li>
+    Dignissim et lorem mollis
+    <ul>
+      <li>Duis eget tincidunt sapien</li>
+      <li>Phasellus viverra ipsum sed</li>
+      <li>
+        Donec pulvinar aliquet turpis
+        <ul>
+          <li>Praesent vitae augue sed</li>
+          <li>Vitae convallis enim lobortis</li>
+          <li>Nisi condimentum ultricies nec</li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+</ul>
+<h2>Ordered List Example</h2>
+<ul>
+  <li>Nulla cursus sem nulla</li>
+  <li>Sed egestas lorem pharetra</li>
+  <li>Pellentesque habitant morbi</li>
+</ul>
+<h2>Ordered List /w Nested Lists Example</h2>
+<ol>
+  <li>Proin consectetur pharetra</li>
+  <li>Aenean lobortis porta dictum</li>
+  <li>
+    Vestibulum in tortor vulputate
+    <ol>
+      <li>Fusce iaculis ante ac diam</li>
+      <li>Ac elementum nisi eleifend</li>
+      <li>
+        Maecenas nec convallis orci
+        <ol>
+          <li>Praesent ut felis quis</li>
+          <li>Etiam quis placerat neque</li>
+          <li>Nunc malesuada luctus varius.</li>
+        </ol>
+      </li>
+    </ol>
+  </li>
+</ol>
+<h2>Mixed List</h2>
+<ul>
+  <li>Proin at pharetra ligula</li>
+  <li>Integer id diam eleifend ac</li>
+  <li>
+    Sit amet blandit tortor dignissim
+    <ol>
+      <li>Consectetur adipiscing elit</li>
+      <li>Mauris fringilla sodales nulla</li>
+      <li>
+        In pulvinar massa ac porttitor
+        <ul>
+          <li>Proin at pharetra ligula</li>
+          <li>Suscipit erat a, aliquet nisi</li>
+          <li>Curabitur tempus dolor massa</li>
+        </ul>
+      </li>
+    </ol>
+  </li>
+</ul>
+<h2>Text Overflow in a List</h2>
+<ul>
+  <li>
+    Proin at pharetra ligula, donec ullamcorper, risus id venenatis. Integer id diam eleifend ac,
+    consectetur adipiscing elit. Fusce iaculis ante ac diam ac elementum nisi eleifend in pulvinar
+    massa ac porttitor.
+  </li>
+</ul>
+<h2>Opt-Out Default Style</h2>
+<p>
+  If you have a menu, built with a <code>&lt;ul&gt;</code>, you might want to opt-out the default
+  styling to avoid getting bullets in your menu with <code>display: none</code>. You can also
+  opt-out of the 8px margin that's added to the top of every <code>&lt;li&gt;</code> element, except
+  the first, with <code>margin-block-start: revert;</code>.
+</p>
```

Comment:
```suggestion
<p>
  In some cases you may not want the default styling. A common example is a navigation menu where an
  HTML list is used but styled to not look like a list. You’re probably already overriding the
  default user agent styles for lists. In addition you may want to override some or all of the Kirby
  default list styling. Which declarations should be made could differ depending on the specific
  case.
</p>
```

Reply:
And the add the following (with examples in code viewers):

Reply:
To remove the custom markers (bullets):

```
li::before { 
  display: none; 
}
```

To remove or control the space between list items you’ll want to target list items with sufficiently high specificity, e.g.:

```
// Selector specificity needs to be equal to or higher than:
// li li, 
// li + li { ... }

ol li,
ul li {
  margin-block-start: revert; // you may want to set a different value
}
```

You may also want to change the `display` and `line-height` properties on list items `li`. Which values to use will depend on the specific case.